docs/content/about-iota/iota-wallet-dashboard/how-to/basics.mdx

```diff
@@ -0,0 +1,82 @@
+---
+description: A collection of how-to guides for managing and using the IOTA Wallet, including balance viewing, sending assets, and security features.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet-dashboard]
+---
+# Basics
+
+You can use the IOTA Wallet Dashboard for anything you would use your [IOTA
+Wallet](../../iota-wallet/getting-started.mdx), as well [migrating](../how-to/migration.mdx) and [managing
+assets](../how-to/assets.mdx), and [managing your vested tokens](../how-to/vesting.mdx).
+
+## Assets and Activity Overview
+
+After you have [connected your wallet](../getting-started.mdx#connect-your-wallet), you will see an overview of your
+coins and assets, as well as recent activity. 
+
+   ![Dashboard overview](/img/about-iota/iota-wallet-dashboard/how-to/basics/dashboard-overview.png)
+
+## Send Coins
+
+To send IOTA or other tokens to another address, follow these steps:
+
+1. Go to the `Home` screen and click the `Send` icon next to under IOTA Balance.
+2. Choose the token you wish to send from the dropdown list.
+3. Enter the amount you want to transfer.
+4. Input the recipient’s address and click `Review` to confirm.
+
+   ![Review](/img/about-iota/iota-wallet-dashboard/how-to/basics/enter-transaction-details.png)
+
+5. Once you verify the transaction details, click `Send Now ->` to complete the transfer.
+
+   ![Send Review](/img/about-iota/iota-wallet-dashboard/how-to/basics/send-now.png)
+
+## Receive Coins
+
+To get your active address details so you can receive coins, clikc on the `Receive` button in the Dashboard's home
+screen.
+
+   ![Dashboard overview](/img/about-iota/iota-wallet-dashboard/how-to/basics/dashboard-overview.png)
+
+The Dashboard will prompt you with a QR code you can use, as well as the address in plain text, and a button to copy the
+address.
+
+   ![Dashboard overview](/img/about-iota/iota-wallet-dashboard/how-to/basics/receive.png)
+
+## View Recent Transaction Details
+
+### From the Dashboard
+
+1. Select a recent transaction from the `Activity` list.
+
+   ![Review activity](/img/about-iota/iota-wallet-dashboard/how-to/basics/select-transaction.png)
+
+2. Click on the transaction to view its details. You can also click on `View on Explorer` to see the transaction in the
```

Comment:
there seems to be an issue rendering this link 
![image](https://github.com/user-attachments/assets/45d4df95-ce3e-4a17-9268-60601a269967). It happens in other places in this page